src/ops/direnv/mod.rs

```diff
@@ -15,8 +17,24 @@ pub fn main(project: &Project) -> OpResult {
     let mut shell_root = project.gc_root_path().unwrap();
     shell_root.push("build-0"); // !!!
 
+    // TODO: don’t start build/evaluation automatically, let the user decide
+    if let Ok(client) = client::ping(DEFAULT_READ_TIMEOUT).connect(
+        &::socket::path::SocketPath::from(::ops::get_paths()?.daemon_socket_file()),
+    ) {
+        client
+            .write(&Ping {
+                nix_file: project.expression().clone(),
+            })
+            .unwrap();
+    } else {
+        eprintln!("Uh oh, your lorri daemon is not running.");
+    }
+
+    // TODO: this needs to move to a daemon message
     if !shell_root.exists() {
-        return ExitError::errmsg("Please run 'lorri watch' before using direnv integration.");
+        return Err(ExitError::errmsg(
+            "Please run 'lorri watch' before using direnv integration.",
```

Comment:
```suggestion
            "Please start `lorri daemon` or run `lorri watch` before using direnv integration.",
```

Reply:
Yeah, this is definitely needed. The UX currently is quite confusing:

```
[grahamc@Petunia:~/projects/github.com]$ cd NixOS/ofborg
direnv: loading .envrc
Uh oh, your lorri daemon is not running.
Please run 'lorri watch' before using direnv integration.

[grahamc@Petunia:~/projects/github.com/NixOS/ofborg]$ cd ..
direnv: unloading

   <--- here I go start `lorri daemon` elsewhere -->

[grahamc@Petunia:~/projects/github.com/NixOS]$ cd ofborg
direnv: loading .envrc
Please run 'lorri watch' before using direnv integration.

    <!-- lorri daemon starts processing -->

[grahamc@Petunia:~/projects/github.com/NixOS/ofborg]$ cd ..
```

Reply:
Can we move this to a different PR?

Reply:
Ok!

Reply:
Actually, please just accept my suggested change as-is since it is an improvement at least. The rest can be another PR

Reply:
Done, issue at https://github.com/target/lorri/issues/99